README.md

```diff
@@ -143,6 +143,10 @@ guide](https://github.com/airbnb/javascript/blob/7684892951ef663e1c4e62ad57d662e
 npx react-codemod sort-comp <path>
 ```
 
+#### `update-react-imports`
+
+Removes unused react imports as a result of a [new Babel transform](https://babeljs.io/blog/2020/03/16/7.9.0#a-new-jsx-transform-11154-https-githubcom-babel-babel-pull-11154) that auto imports React for JSX and converts default imports (`import React from 'react'`) to named imports (`import * as React from 'react`). This codemod is intended for anyone using `runtime: automatic` in `@babel/preset-react` or `@babel/plugin-transform-react-jsx`.
```

Comment:
I took a shot at rewriting this. Saying 'new babel transform' is odd because it won't be new in the future, but this will still be useful. Feel free to change as you please!
```suggestion
[As of Babel 7.9.0]((https://babeljs.io/blog/2020/03/16/7.9.0#a-new-jsx-transform-11154-https-githubcom-babel-babel-pull-11154), when using `runtime: automatic` in `@babel/preset-react` or `@babel/plugin-transform-react-jsx`, you will not need to explicitly import React for compiling jsx. This codemod removes the redundant import statements. ```